packages/core/src/metadata.ts

```diff
@@ -2,3 +2,11 @@ export type Metadata = {
   name: string;
   description?: string;
 };
+
+export type ComponentMetadata = Metadata & {
+  isDraggable: boolean;
+  isResizable: boolean;
+  displayName: string;
```

Comment:
I think our version field `category/name` already provides a unique name for a component. So in the spec aspect, shall we tell end-users when and how to set displayName?

Reply:
In my opinion, the category/name is for programming, not for display. For example, GridLayout's name is `grid_layout`, but maybe we want display it as `Grid Layout` or something else. I think declare it in spec is more convinient than somewhere else.

Reply:
fair enough